src/cli.js

```diff
@@ -11,10 +11,52 @@ import fs from 'fs';
 import ciChoices from '../ci-choices.json';
 import containerizationChoices from '../containerization-choices.json';
 
+function initDocker(config) {
```

Comment:
move this to a new file

Reply:
If we move the string literal to a template file, I don't think moving this method to a new file would be necessary.

Reply:
just for sanity sake we should move it to a new file, eventually we could have more tools, and could take advantage of ES6 classes and OOO